docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -49,7 +49,7 @@ Depending on the use-case and the `params` you pass to the `create` method, it c
+| `strategy?` | `string` | Select the first factor verification strategy. The `strategy` value depends on the object's `identifier` value. Each authentication identifier supports different verification strategies.<br/>Possible `strategy` values are: <ul><li>`email_link`: User will receive an email magic link via email. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`email_code`: User will receive a one-time authentication code via email. At least one email address should be on file for the user. The `email_address_id` parameter can also be specified to select one of the user's known email addresses.</li><li>`phone_code`: User will receive a one-time authentication code in their phone, via SMS. At least one phone number should be on file for the user. The `phone_number_id` parameter can also be specified to select which of the user's known phone numbers the SMS will go to.</li><li>`web3_metamask_signature`: The verification will attempt to be completed using the user's web3 wallet public address. The `web3_wallet_id` parameter can also be specified to select which of the user's known web3 wallets will be used. Currently Clerk supports [Metamask](https://metamask.io/).</li><li>`oauth_<provider>`: The user will be authenticated with their Social login account. See available [OAuth Strategies](/docs/references/javascript/types/oauth-strategy).</li><li>`ticket`: The user will be authenticated via the ticket *or token* generated from the Backend API.</li></ul> |
```

Comment:
This diff is atrocious, thanks GitHub lmao

Reply:
yeah.. no clue why that happened lmao